smack-extensions/src/main/java/org/jivesoftware/smackx/usertune/UserTune.java

```diff
@@ -0,0 +1,82 @@
+/**
+ *
+ * Copyright 2019 Aditya Borikar.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smackx.usertune;
+
+/**
+ * Child elements described as in XEP-0118: User Tune.
+ * @see <a href="https://xmpp.org/extensions/xep-0118.html#table-1"> </a>
+ *
+ * TODO: If the user is listening to a stream, length should not be included.
+ */
+public class UserTune {
+
+    String artist;
+    int length;
+    int rating;
+    String source;
+    String title;
+    int track;
+    String uri;
+
+    public String getArtist() {
+        return artist;
+    }
+    public void setArtist(String artist) {
```

Comment:
If you have final members and make use of the Builder pattern, you dont need setters in the class any more.